### PR TITLE
Registration: add a line for registering against the proxy

### PR DESCRIPTION
**Pull request by admd**
This page is also in good condition. It was just missing a line about the proxy field in case, the user wants to register the system against it.

### CODE REVIEW

**Comment by keichwa** [comment]
Thanks for noting, Abid!  We probably must indicate that this is "optional".  I'm sure @Loquacity will find good wording!

File: modules/client-configuration/pages/registration-webui.adoc [hunk 1]
```diff
@@ -14,6 +14,7 @@ By default, the SSH  port is [systemitem]``22``.
 By default, the username is [systemitem]``root``.
 . In the [guimenu]``Password`` field, type password to log in to the client.
 . In the [guimenu]``Activation Key`` field, select the activation key that is associated with the software channel you want to use to bootstrap the client.
+. In the [guimenu]``Proxy`` field, select the proxy from the available ones to register the system against it.
```
Comment (Loquacity) [inline modules/client-configuration/pages/registration-webui.adoc]:
```suggestion
. OPTIONAL: In the [guimenu]``Proxy`` field, select which proxy to register the client against.
```

Comment (Loquacity) [reply modules/client-configuration/pages/registration-webui.adoc]: :question:

Comment (keichwa) [reply modules/client-configuration/pages/registration-webui.adoc]: yes, but ... I have the feeling that ,ost of the time we say "register to", "against" seems to occur 6 x.

Comment (Loquacity) [reply modules/client-configuration/pages/registration-webui.adoc]: You're right. I'll edit.

**Comment by admd** [comment]
Thank you @Loquacity & @keichwa. I have changed the line according to the suggestions.

**Pull request merged**